sass/popup.scss

```diff
@@ -586,8 +586,67 @@ svg {
     margin: 10px 0 0 10px;
   }
 
+  .prefrence-item {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .prefrence-item select {
+    width: 100px;
+    padding: 5px 10px;
+    @include themify($themes) {
+      border: themed("grey-2") 1px solid;
+      background: themed("white-1");
+      color: themed("black-1");
+    }
+  }
+
+  .prefrence-item input[type="checkbox"] {
+    position: relative;
+    border: none;
+    background: none;
+    width: 34px;
+    height: 16px;
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+    outline: none;
+  }
+
+  .prefrence-item input[type="checkbox"]::before {
```

Comment:
The fact that you can do this in pure CSS is really cool

Reply:
😄